### PR TITLE
Fix CI Tests and Artifact Cache Issues

### PR DESCRIPTION
**Pull request by fchorney**
Use CLI script to automatically replace the 1.3 test with a 1.5 test (if necessary), as well as use new cache steps to cache the julia build artifacts

### CODE REVIEW

File: .github/workflows/CI.yml [hunk 1]
```diff
@@ -15,46 +15,44 @@ jobs:
...
         include:
-          # Add a 1.0 job just to make sure we still support it
-          - os: ubuntu-latest
-            version: 1.0.5
-            arch: x64
-          # Add a 1.3 job because that's what Invenia actually uses
+          # Add a 1.5 job because that's what Invenia actually uses
           - os: ubuntu-latest
-            version: 1.3
+            version: 1.5
```
Comment (omus) [inline .github/workflows/CI.yml:36]:
```suggestion
            version: "1.5"
```

**Review by omus — APPROVED** [review]

**Comment by omus** [comment]
I've asked @mjamei19 to disable Appveyor

**Review by mattBrzezinski — APPROVED** [review]

**Comment by omus** [comment]
I retriggered the CI as I wanted to see if Appveyor was disabled properly. It appears not.

**Comment by fchorney** [comment]
> I retriggered the CI as I wanted to see if Appveyor was disabled properly. It appears not.

I think I figured it out.

Settings -> Branch -> edit Master -> Turn off required for the old travis/appveyor jobs

**Pull request merged**